qutebrowser/browser/webkit/network/networkmanager.py

```diff
@@ -408,6 +400,14 @@ def createRequest(self, op, req, outgoing_data):
                 # the webpage shutdown here.
                 current_url = QUrl()
```

Comment:
Are `QUrl()` and `None` equivalent for per domain settings? If there is, there will be a descrepancy between this and when the method is called with missing args (if it is in the future).

Reply:
Ufff, you are asking the right question.
I hope someone knows^^

Reply:
Good catch, it looks like they [are treated differently](https://github.com/Ongy/qutebrowser/blob/40b2d03688ab14703fe8eae6f8d98e578e623995/qutebrowser/config/configutils.py#L158) and an empty `QUrl` is apparently invalid so a `QtValueError` will be thrown from [here](https://github.com/Ongy/qutebrowser/blob/40b2d03688ab14703fe8eae6f8d98e578e623995/qutebrowser/utils/urlmatch.py#L278).

Reply:
Good catch! Fixed in c6dc9206e6cdedc74a401fc2827c1f855b945805.